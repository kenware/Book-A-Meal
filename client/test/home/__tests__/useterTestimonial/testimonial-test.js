import React from 'react';
import { shallow } from 'enzyme';
import Testimonial from '../../../../components/home/userTestimonial';

describe('Test user testimonial components', () => {
  it('renders correctly', () => {
    const tree = shallow(<Testimonial />);
    expect(tree).toMatchSnapshot();
  });

  it('should have one testimonial-wrap class as a container', () => {
    const tree = shallow(<Testimonial />);
    expect(tree.find('.testimonial-wrap')).toHaveLength(1);
  });

  it('should have two user-testimonial-menu class container', () => {
    const tree = shallow(<Testimonial />);
    expect(tree.find('.user-testimonial-menu').length).toEqual(2);
  });
  it('should have two img tag', () => {
    const tree = shallow(<Testimonial />);
    expect(tree.find('img').length).toEqual(2);
  });
  it('should have two img container class', () => {
    const tree = shallow(<Testimonial />);
    expect(tree.find('.container').length).toEqual(2);
  });
  it('should have 4 p tag', () => {
    const tree = shallow(<Testimonial />);
    expect(tree.find('p').length).toEqual(4);
  });
  it('should have 9 div', () => {
    const tree = shallow(<Testimonial />);
    expect(tree.find('div').length).toEqual(9);
  });
  it('should have three h', () => {
    const tree = shallow(<Testimonial />);
    expect(tree.find('h2').length).toEqual(2);
  });
  it('it should have two container class', () => {
    const tree = shallow(<Testimonial />);
    expect(tree.find('.container')).toHaveLength(2);
  });
});
