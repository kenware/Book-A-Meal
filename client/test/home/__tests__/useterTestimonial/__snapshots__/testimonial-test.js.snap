// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test user testimonial components renders correctly 1`] = `
<div
  className="testimonial-wrap"
>
  <div>
    <div
      className="user-testimonial-menu"
    >
      <div
        className="testimonial-photo"
      >
        <img
          alt="profile"
          className="rounded-circle"
          src="image/eze.jpg"
          style={
            Object {
              "height": "100px",
              "width": "100px",
            }
          }
        />
      </div>
      <p
        className="justify text-pad"
      >
        Book-A-Meal is just amazing!. For the first time i can book for any meal by a single click any where around the world, without having to go to any  resturant .This is really awesome.
      </p>
      <div
        className="container"
      >
        <h2
          className="white-color"
        >
          Kevin
        </h2>
        <p
          className="text-center white-color"
        >
          Developer
        </p>
      </div>
    </div>
  </div>
  <div>
    <div
      className="user-testimonial-menu"
    >
      <div
        className="testimonial-photo"
      >
        <img
          alt="profile"
          className="rounded-circle"
          src="image/face1.jpg"
          style={
            Object {
              "height": "100px",
              "width": "100px",
            }
          }
        />
      </div>
      <p
        className="justify l-r-pad-text"
      >
        Book-A-Meal is  awesome!.This have preventd the stress of having to go down the street from one resturant to another, i have my favorite delivered to my door step b a single click.This is really awsome.
      </p>
      <div
        className="container"
      >
        <h2
          className="white-color"
        >
          Jessica Jones
        </h2>
        <p
          className="white-color text-center"
        >
          Actor
        </p>
      </div>
    </div>
  </div>
</div>
`;
